share/revng/test/configuration/revng-c/name-collisions.yml

```diff
@@ -0,0 +1,32 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+tags:
+  - name: name-collisions
+sources:
+  - tags: [name-collisions]
+    prefix: share/revng/test/tests/name-collisions/
+    members:
+      - no-collisions
+      - function-name
+      - argument-name
+      - custom-prefix
+
+commands:
+  - type: revng-c.name-collisions
+    from:
+      - type: source
+        filter: name-collisions
+    suffix: /
+    command: |-
+      revng model opt -Y -verify "${INPUT}/input-model.yml"
+        1> "${OUTPUT}/model.yml"
+        2> "${OUTPUT}/error.txt"
+
+      && revng artifact emit-model-header /dev/null --model "${OUTPUT}/model.yml"
+        | revng ptml | tee "${OUTPUT}/header.h"
```

Comment:
I think this came up a lot a lot in the past - this helps debugging quite a bit - since you don't need to partially rerun stuff - outputs are always there. Especially when debugging non-deterministic problems (which caused my to `tee` it).

But I guess if you're adamant about it, it can be dropped now that the bug is fixed.

Reply:
OK, but it's a bit hard to figure out what the outputs are, given they are non-standard (as opposed to a `--resume` directory, for instance). Maybe add a comment to describe the layout of the output directory.

Reply:
If you run the test it is, if you read its source not so much.

```
# This test produces three files for debugging purposes: ...
```

Reply:
I guess that makes sense. Will do!

Reply:
Done, please take a look!